### ISSUE TITLE
Force output size or minimum size option

### ISSUE THREAD

**Issue by wonsuc** [zetbaitsu/Compressor#41]
My application have to compress image to defined size (1080 * 1080)
Even the original image size is smaller than desired size I have to scale it to large resolution.
So it would be nice if it has "Force size" or "Minimum size" option.
Thanks.

**Comment by leleliu008** [comment]
Yes, I think need minWidth、minHeight、maxWidth、maxHeight、format、maxFileSize options
